core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -105,6 +107,35 @@ public void testPorts() {
     }
   }
 
+  @Test
+  public void testJson() {
+    assertFalse(TSERV_COMPACTION_SERVICE_META_EXECUTORS.getType().isValidFormat("notJson"));
```

Comment:
Do we still need this test for this specific property since this is tested on line 134?
`invalids.forEach(j -> assertFalse(prop.getType().isValidFormat(j))); `

Reply:
removed with 7fa8acabb7 (and some additional cases added)